### PR TITLE
Migrate project_setup and oss-fuzz related cron jobs.

### PR DESCRIPTION
**Pull request by hogo6002**
1. Moved cron from internal/bot/tasks to internal/
2. Modified triage GAE
3. Added project_setup and oss-fuzz related cron jobs.

### CODE REVIEW

**Review by Alan32Liu — APPROVED** [review]

File: src/appengine/handlers/cron/triage.py [hunk 1]
```diff
@@ -13,478 +13,16 @@
...
-          count_per_project + 1)
-      return False
-
-    logs.log(
-        f'Skipping bug filing for {testcase.key.id()} as it is throttled.\n'
-        f'{count_per_project} bugs have been filed from '
-        f'{self._bug_throttling_cutoff} '
-        f'to {datetime.datetime.now()} for project {testcase.project_name}')
-    return True
+    triage_module = importlib.import_module('clusterfuzz._internal.cron.triage')
```
Comment (oliverchang) [inline src/appengine/handlers/cron/triage.py:27]: Can we just import this directly without having to rely on importlib?

File: src/clusterfuzz/_internal/cron/libs/helpers.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""helper.py is a kitchen sink. It contains static methods that are used by
+   multiple handlers."""
+
+import sys
+import traceback
+
+
+class EarlyExitError(Exception):
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/cron/libs/helpers.py:21]: nit: we can just delete this one completely.

File: src/clusterfuzz/_internal/cron/oss_fuzz_build_status.py [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+
+def main():
+  """Build status checker."""
+  for build_type, status_url in BUILD_STATUS_MAPPINGS:
+    try:
+      response = requests.get(status_url, timeout=HTTP_GET_TIMEOUT_SECS)
+      response.raise_for_status()
+      build_status = json.loads(response.text)
+    except (requests.exceptions.RequestException, ValueError) as e:
+      raise helpers.EarlyExitError(str(e), response.status_code)
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/cron/oss_fuzz_build_status.py:294]: We can just replace this with a RuntimeError(str).

**Review by oliverchang — APPROVED** [review]
nice, thanks! just some minor comments

File: src/clusterfuzz/_internal/cron/cleanup.py [hunk 4]
```diff
@@ -23,10 +23,10 @@
 from clusterfuzz._internal.base import errors
 from clusterfuzz._internal.base import memoize
 from clusterfuzz._internal.base import utils
-from clusterfuzz._internal.bot.tasks.cron.libs import mail
 from clusterfuzz._internal.chrome import build_info
 from clusterfuzz._internal.crash_analysis import crash_comparer
 from clusterfuzz._internal.crash_analysis import severity_analyzer
+from clusterfuzz._internal.cron.libs import mail
```
Comment (jonathanmetzman) [inline src/clusterfuzz/_internal/cron/cleanup.py:29]: Thanks!

**Review by jonathanmetzman — APPROVED** [review]
LGTM

**Review by oliverchang — APPROVED** [review]

**Pull request merged**